2008_Flights_7M/omnisci_ddl/flights_2008_7M.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE flights_2008_7M (
+  flight_year SMALLINT,
+  flight_month SMALLINT,
+  flight_dayofmonth SMALLINT,
+  flight_dayofweek SMALLINT,
+  deptime SMALLINT,
+  crsdeptime SMALLINT,
+  arrtime SMALLINT,
+  crsarrtime SMALLINT,
+  uniquecarrier TEXT ENCODING DICT(32),
+  flightnum SMALLINT,
+  tailnum TEXT ENCODING DICT(32),
+  actualelapsedtime SMALLINT,
+  crselapsedtime SMALLINT,
+  airtime SMALLINT,
+  arrdelay SMALLINT,
+  depdelay SMALLINT,
+  origin TEXT ENCODING DICT(32),
+  dest TEXT ENCODING DICT(32),
+  distance SMALLINT,
+  taxiin SMALLINT,
+  taxiout SMALLINT,
+  cancelled SMALLINT,
+  cancellationcode TEXT ENCODING DICT(32),
+  diverted SMALLINT,
+  carrierdelay SMALLINT,
+  weatherdelay SMALLINT,
+  nasdelay SMALLINT,
+  securitydelay SMALLINT,
+  lateaircraftdelay SMALLINT,
+  dep_timestamp TIMESTAMP(0),
+  arr_timestamp TIMESTAMP(0),
+  carrier_name TEXT ENCODING DICT(32),
+  plane_type TEXT ENCODING DICT(32),
+  plane_manufacturer TEXT ENCODING DICT(32),
+  plane_issue_date DATE ENCODING DAYS(32),
+  plane_model TEXT ENCODING DICT(32),
+  plane_status TEXT ENCODING DICT(32),
+  plane_aircraft_type TEXT ENCODING DICT(32),
+  plane_engine_type TEXT ENCODING DICT(32),
+  plane_year SMALLINT,
+  origin_name TEXT ENCODING DICT(32),
+  origin_city TEXT ENCODING DICT(32),
+  origin_state TEXT ENCODING DICT(32),
+  origin_country TEXT ENCODING DICT(32),
+  origin_lat FLOAT,
+  origin_lon FLOAT,
+  dest_name TEXT ENCODING DICT(32),
+  dest_city TEXT ENCODING DICT(32),
+  dest_state TEXT ENCODING DICT(32),
+  dest_country TEXT ENCODING DICT(32),
+  dest_lat FLOAT,
+  dest_lon FLOAT,
+  origin_merc_x FLOAT,
+  origin_merc_y FLOAT,
+  dest_merc_x FLOAT,
+  dest_merc_y FLOAT)
+WITH (FRAGMENT_SIZE=2000000);
+
+copy flights_2008_7M from 's3://omnisci-community/datasets/airline_on-time_performance/flights_2008_7M.csv.gz' with (header='true');
```

Comment:
Missing newline at end of file, not sure if omnisql will always accept that.

Reply:
added newline at the end.